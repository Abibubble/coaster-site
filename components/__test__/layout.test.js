import React from 'react'
import renderer from 'react-test-renderer'
import Layout from '../layout'

describe('Layout', () => {
  it('should render layout component', () => {
    const tree = renderer.create(<Layout>Layout</Layout>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
