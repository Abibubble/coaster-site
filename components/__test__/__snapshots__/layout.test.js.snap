// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render layout component 1`] = `
Array [
  <header
    className="header"
  >
    <div
      className="desktopHeader"
    >
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <div
          className="logoContainer"
        >
          <p>
            The First Drop logo
          </p>
        </div>
      </a>
      <div
        className="search"
      >
        <form
          onSubmit={[Function]}
        >
          <input
            onChange={[Function]}
            placeholder="Search"
            type="text"
            value=""
          />
        </form>
      </div>
      <nav
        className="mainNav"
      >
        <ul>
          <li>
            <a
              href="/all-coasters"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              All Coasters
            </a>
          </li>
          <li>
            <a
              href="/all-parks"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              All Parks
            </a>
          </li>
          <li>
            <a
              href="/profile"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Profile
            </a>
          </li>
          <li>
            <a
              href="/my-coasters"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              My Coasters
            </a>
          </li>
          <li>
            <a
              href="/my-parks"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              My Parks
            </a>
          </li>
          <li>
            <a
              href="/wish-list"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Wish List
            </a>
          </li>
          <li>
            <a
              href="/submit"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Submit
            </a>
          </li>
          <li>
            <a
              href="/logout"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Logout
            </a>
          </li>
          <li>
            <a
              href="/login"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Login
            </a>
          </li>
        </ul>
      </nav>
    </div>
    <div
      className="mobileHeader"
    >
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <div
          className="logoContainer"
        >
          <p>
            The First Drop logo
          </p>
        </div>
      </a>
      <button
        className="burgerIconContainer"
        onClick={[Function]}
      >
        <svg
          className="burgerIcon"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 448 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
          />
        </svg>
      </button>
      <nav
        className="hiddenMainNav"
      >
        <ul>
          <li>
            <a
              href="/all-coasters"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              All Coasters
            </a>
          </li>
          <li>
            <a
              href="/all-parks"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              All Parks
            </a>
          </li>
          <li>
            <a
              href="/profile"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Profile
            </a>
          </li>
          <li>
            <a
              href="/my-coasters"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              My Coasters
            </a>
          </li>
          <li>
            <a
              href="/my-parks"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              My Parks
            </a>
          </li>
          <li>
            <a
              href="/wish-list"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Wish List
            </a>
          </li>
          <li>
            <a
              href="/submit"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Submit
            </a>
          </li>
          <li>
            <a
              href="/logout"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Logout
            </a>
          </li>
          <li>
            <a
              href="/login"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Login
            </a>
          </li>
          <li>
            <div
              className="search"
            >
              <form
                onSubmit={[Function]}
              >
                <input
                  onChange={[Function]}
                  placeholder="Search"
                  type="text"
                  value=""
                />
              </form>
            </div>
          </li>
        </ul>
      </nav>
    </div>
  </header>,
  <main
    className="main"
  >
    <div
      className="content"
    >
      Layout
    </div>
  </main>,
  <footer
    className="main"
  >
    <p>
      Footer
    </p>
  </footer>,
]
`;
